modules/sandboxed-containers-metrics-list.adoc

```diff
@@ -0,0 +1,16 @@
+// Module is included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-metrics-list_{context}"]
+= About {sandboxed-containers-first} metrics
+
+{sandboxed-containers-first} metrics enable administrators to monitor how their sandboxed containers are running.
+
+{sandboxed-containers-first} metrics are collected for the following categories:
+
+Kata agent metrics:: Kata agent metrics display information about the kata agent process running in your sandboxed containers. These metrics include data from `/proc/<pid>/[io, stat, status]`.
```

Comment:
Maybe too detailed, but : The agent actually runs inside the VM.
We could phrase it like :
"[...] the kata agent process running in the VM embedded in your sandboxed containers."
(I'm fine leaving it if we don't want to go into too much details)